source/bin/tvm.rs

```diff
@@ -45,14 +45,15 @@ enum ExitCode {
     InvalidOption,
     MissingFile,
     InvalidFile,
+    MultipleFiles,
```

Comment:
Well, that is the point, imagine someone actually passing two files.

`Usage: tvm [options] [file]`

That means that passing arguments is not currently supported ⇒ Error when passing multiple options.